modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -133,6 +134,50 @@ public void testAddObjectRelationship() throws Exception {
 		Assert.assertFalse(_hasTable(objectRelationship.getDBTableName()));
 	}
 
+	@Test
+	public void testAddObjectRelationshipMappingTableValues() throws Exception {
+		String name = StringUtil.randomId();
```

Comment:
The variable name is only used 1 time. Could you put the code directly where it is going to be used, please?